docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md

```diff
@@ -0,0 +1,228 @@
+granular available per grading period — typically a unit smaller than a
+trimester or semester.
+
+The fact tables do not support the gradebook for two reasons. First, there are
+too many ways to calculate total grade from gradebook entries. Second, field
+experience shows that Ed-Fi implementations are much more likely to populate
+the `Grade` table (by grading period) than the `GradeBookEntry` (which contains
+homework, quiz, and test grades).
+
+While there is no independent `Section` view, the `SectionKey` value may be
+critical for [user data
+authorization](../row-level-security-collection/readme.md).
+
+![StudentEarlyWarningFact](https://edfidocs.blob.core.windows.net/$web/img/reference/analytics-middle-tier/student-early-warning-fact.jpg)
+
+The early warning attendance and behavior data are supplied
+by `StudentEarlyWarningFact`, using three key fields: Student, School, Date.
+The `SchoolKey` supports students who might be dual-enrolled and it may be used
+for user data authorization.
+
+Attendance rates will not be pre-calculated due to the variation in business
+rules. Instead, the view provides multiple views on attendance, with data from
+two different perspectives:
+
+1. Attendance indicators
+    * Tardy
+    * Excused Absence
+    * Unexcused Absence
+    * Present
+2. Attendance scope
+    * Homeroom
+    * Any Class
+    * School
+
+Which factors are used for calculating attendance depends on the school and on
+how attendance is recorded in the source systems. Some systems may record
+attendance in the positive sense (student is present), while others might only
+record when the student is absent or late.
+
+The behavior fields provide counts of infractions per day. Not all behavior
+incidents are recorded here, only those marked as a "state offense" or as a
+"code of conduct violation."
+
+## Installation
+
+Install using the option code "EWS"
+
+```powershell
+.\EdFi.AnalyticsMiddleTier.Console.exe -c "..." -o EWS
+```
+
+For more information, see the [AMT Deployment
+Guide](../../../deployment-guide).
+
+## Configuration
+
+If your schools use letter grades instead of numeric grades, in some or all
+cases, then you may wish to review the default mapping from letter to numeric
+grades as used by the `StudentSectionGradeFact` view in the Early Warning System
+collection.
+
+This mapping is provided so that analyses can run aggregate calculations over
+numeric values that would not otherwise be available. Modify or add rows as
+needed to the `[analytics_config].[LetterGradeTranslation]`. Default values are:
+
+| Letter | Numeric |
+| --- | --- |
+| A   | 95.0 |
+| B   | 85.0 |
+| C   | 75.0 |
+| D   | 65.0 |
+| F   | 55.0 |
+
+That is, a grade of A will be treated a 95.0, a grade of B will be treated as
+85.0, and so forth.
+
+## Sample Scripts
+
+The source code repository contains a number of sample scripts in the **[samples
+directory](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier/tree/main/samples)**.
+
```

Comment:
This directory is not valid. I tried searching and found this. https://github.com/Ed-Fi-Alliance/Ed-Fi-Analytics-Middle-Tier/tree/develop/samples

Reply:
Your link is to the pre-open source version. I'm not having any problems with this link.